.github/component_owners.yml

```diff
@@ -6,6 +6,8 @@ components:
   src/OpenTelemetry.Exporter.Geneva/:
     - codeblanch
     - rajkumar-rangaraj
+    - xiang17
+    - TimothyMothra
```

Comment:
nit: Alphabetical ordering here and in README list

Reply:
adjusted